projects/lila-finance/index.js

```diff
@@ -5,7 +5,10 @@ async function tvl(_, _b, _cb, { api, }) {
   data = data.filter(i => i.strategy !== '0x0000000000000000000000000000000000000000')
   const aTokens = await api.multiCall({  abi: 'address:aToken', calls: data.map(i => i.strategy)})
   const ownerTokens = data.map((i, idx) => [[i.asset, aTokens[idx]], i.strategy])
-  return api.sumTokens({ ownerTokens })
+  const aTokenSum  = await api.sumTokens({ ownerTokens });
+  const TrueTokenSum = {};
+  ownerTokens.forEach(([[a, b], _]) => { TrueTokenSum['arbitrum:'+a] = aTokenSum['arbitrum:'+b.toLowerCase()]; });
```

Comment:
issue right now is, radiant wSTETH token is not getting priced, but I have added a fix on the coins repo for it, so this PR is not not needed: https://github.com/DefiLlama/defillama-server/commit/9ef41bcdf52dd98f03eddde4eb140c15036c7fb1